crates/rpc/src/v02/method/get_state_update.rs

```diff
@@ -439,18 +365,22 @@ mod tests {
     /// Execute a single test case and check its outcome.
     async fn check(test_case_idx: usize, test_case: &(RpcContext, BlockId, TestCaseHandler)) {
         let (context, block_id, f) = test_case;
-        let result = get_state_update(
+        let mut result = get_state_update(
             context.clone(),
             GetStateUpdateInput {
                 block_id: *block_id,
             },
         )
         .await;
+        if let Ok(r) = result.as_mut() {
+            r.sort();
+        }
```

Comment:
Using hashmaps / hashsets instead of vecs in the RPCs `StateUpdate` would make comparing the values much easier - so we wouldn't need to sort. Didn't want to change even more code, so not done it yet - wdyt?